plugins/services/src/js/service-configuration/ServiceConfigBaseSectionDisplay.js

```diff
@@ -29,6 +30,12 @@ class ServiceConfigBaseSectionDisplay extends React.Component {
     return { values: [] };
   }
 
+  getMountPointName() {
+    throw Error(
+      "You have to implement this method in your class providing a unique MountPointName, Namespace: CreateService:ServiceConfigDisplay:App:"
```

Comment:
We have to throw here, because creating an empty Mountpoint (or creating the same one twice when this method isnt implemented in a subclass), can lead to an infinite loop (sort of).
I have created an Issue for refactoring it (DCOS-19563)